doc/ref_arch/kubernetes/README.md

```diff
@@ -5,18 +5,19 @@
 This is Kubernetes based Reference Architecture (RA-2)
 
 ## Release Information
-**Bundle: _3_**
+**Bundle: _4_**
 
 **Version: _0_**
 
-**Release Date: _15th May 2020_**
+**Release Date: _25th September 2020_**
 
 ## Bundle/Version History
 
-| Bundle.Version | Date | Note
-| --- | --- | --- |
-| 1.0-alpha | 10th January 2020 | Snezka Release|
-| 3.0 | 15th May 2020 | Baldy Release|
+| Bundle.Version    | Date              | Note
+| ---               | ---               | ---               |
+| 1.0-alpha         | 10th January 2020 | Snezka Release    |
+| 3.0               | 15th May 2020     | Baldy Release     |
```

Comment:
why did we jump a release for RA-2? If we make this v2.0 for Baldy, Baraque will only be v3.0 aligned with others.

Reply:
+1

Reply:
we discussed this before though the bundle discussion. for RA-2, we are aligning it with RM latest release.